ans-abstract/abstract.tex

```diff
@@ -49,21 +49,21 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \section{Introduction}
 
-Climate change presents a threat that we should address swiftly. Decarbonizing the electric grid through variable renewable energy and nuclear power seems to be a remedy. Unfortunately, a carbon neutral electric grid will be insufficient to halt climate change because transportation contributes more to \gls{GHG} emissions than electricity. As seen in Figure \ref{fig:ghg}, transportation produced the most \glspl{GHG} in the US in 2017. Thus, decarbonizing transportation underpins global carbon reduction. Accordingly, the \gls{UIUC} has committed to the Illinois Climate Action Plan (iCAP) which aims to attain carbon neutrality by 2050 \cite{noauthor_illlinois_2015}.
+Climate change presents a threat that we should address swiftly. Decarbonizing the electric grid through variable renewable energy and nuclear power seems to be a remedy. Unfortunately, a carbon neutral electric grid will be insufficient to halt climate change because transportation contributes more to \gls{GHG} emissions than electricity. As seen in Figure \ref{fig:ghg}, transportation produced the most \glspl{GHG} in the US in 2017. Thus, decarbonizing transportation underpins global carbon reduction. Accordingly, the \gls{UIUC} has committed to the Illinois Climate Action Plan (iCAP) which aims to attain carbon neutrality by 2050 \cite{university_of_illinois_at_urbana-champaign_illlinois_2015}.
```

Comment:
No change necessary this time, but for the future, I strongly recommend hardwrapping your text at 70 or 80 characters. It makes these changesets readable.

Reply:
Sorry about that! I will be more careful about that in the future.